docs/modules/ROOT/pages/vshn-managed/postgresql/major-upgrade.adoc

```diff
@@ -0,0 +1,53 @@
+= PostgreSQL Major Version Upgrade
+
+== Overview
+VSHNPostgreSQL supports major version upgrades up to the latest PostgreSQL 17.
+
+[WARNING]
+====
+Carefully follow the steps below to avoid prolonged downtime or data loss.
+====
+
+== Preparing for the Upgrade
+Major version upgrades carry risks, so before proceeding, ensure the following:
+
+. **Check Disk Space**: Although the upgrade process uses hard links for data migration, we recommend having at least double the free storage as a precaution.
```

Comment:
How can I do that as end user? We need to describe that...

Reply:
Added another section at the bottom of the page.